### PR TITLE
Altering the navigation to workaround a selenium issue

### PR DESCRIPTION
**Pull request by blrm**
This hack aims to work around an issue with navigation when setting up workers for replication tests.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 13bce5bc868b12e83ed665bbcae2a69f3227ef22:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/distributed/test_appliance_replication.py [hunk 1]
```diff
@@ -118,6 +118,11 @@ def configure_db_replication(db_address):
     """
     conf.set_replication_worker_host(db_address)
     flash.assert_message_contain("Configuration settings saved for CFME Server")
+    try:
+        sel.force_navigate("cfg_settings_currentserver_server")
+    except Exception:
```
Comment (seandst) [inline cfme/tests/distributed/test_appliance_replication.py]: Should this only be handling [](https://selenium-python.readthedocs.org/en/latest/api.html#selenium.common.exceptions.UnexpectedAlertPresentException)`UnexpectedAlertPresentException`?

**Comment by dajoRH** [comment]
Lint report for commit 1b3fa9669acc99ed0484cc463bb72659ebb94577:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6ebdf1f7dd99fd19b6331ce694d4e95c24a12a8b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b2af4ed6ea677253c30762dcc93d381a7d7acad3:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 293e8f82e1835994870308566efe27499910e5fc:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**